js/util.js

```diff
@@ -51,3 +51,37 @@ export const buildSlug = value => {
  * of the callback function and flattened to a depth of 1.
  */
 export const flatMap = (f, xs) => xs.reduce((acc, x) => acc.concat(f(x)), []);
+
+/**
+ * Parses a query string and builds an object with each
+ * query parameter and its respective value.
+ *
+ * @param {String} query The query string to be parsed.
+ * @returns An object containing the query parameters
+ * keys and its respective values
+ */
+export const unpackQuery = query => {

```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams ?

Reply:
Does not work, related to this: https://github.com/facebook/react-native/issues/23922
TLDR: necessary to add a polyfill